tests/cupyx_tests/scipy_tests/ndimage_tests/test_measurements.py

```diff
@@ -96,3 +98,89 @@ def test_label_swirl(self, xp, scp):
         x = xp.array(x)
         labels, num_features = scp.ndimage.label(x)
         return labels
+
+
+@testing.gpu
+@testing.parameterize({'op': 'sum'}, {'op': 'mean'}, {'op': 'variance'},
+                      {'op': 'standard_deviation'})
```

Comment:
You can use `testing.product` helper.
```suggestion
@testing.parameterize(*testing.product({
    'op': ['sum', 'mean', 'variance', 'standard_deviation'],
}))
```